Tools/Algorithms/stencil.py

```diff
@@ -5,13 +5,16 @@
 measure is not larger than the error threshold).
 Reference: https://doi.org/10.1016/j.cpc.2022.108457
 
-Run the script simply with "python stencil.py" (or with "run stencil.py"
-using IPython). The user can modify the input parameters set in the main
-function at the end of the file.
+How to run the script:
+$ python stencil.py --path path_to_input_file
```

Comment:
This needs to be updated to use the correct option `--input_file`.

Reply:
Thanks, done in [9fa34fd](https://github.com/ECP-WarpX/WarpX/pull/4016/commits/9fa34fd690cbca654cc2f432e24f8d314ef08760).